Assets/UIElements/_Elements/XRUI_Feedback/Scripts/XRUI_Feedback.cs

```diff
@@ -121,16 +122,19 @@ private void CreateNearCollider()
 
         nearColliderObject.transform.localPosition = Vector3.zero;
         nearColliderObject.transform.localRotation = Quaternion.identity;
+        /// its fix scale on rotation if some axis is diffent 
 
         switch (nearColliderType)
         {
             case NearColliderType.Box:
                 {
+
+                    nearColliderObject.transform.localScale = new Vector3(1f / transform.localScale.x, 1f / transform.localScale.y, 1f / transform.localScale.z);
                     Vector3 boxScale = transform.localScale;
                     boxScale.Scale(nearColliderScale);
 
                     nearCollider = nearColliderObject.AddComponent<BoxCollider>();
-                    ((BoxCollider)nearCollider).center = nearColliderOffset;
+                    ((BoxCollider)nearCollider).center = new Vector3(nearColliderOffset.x * transform.localScale.x, nearColliderOffset.y * transform.localScale.y, nearColliderOffset.z * transform.localScale.z);
```

Comment:
Here it's possible to use Vector3.Scale(). This function multiply two vectors

Reply:
Done